pkgs/development/python-modules/nfstream/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, buildPythonApplication
+, psutil
+, numpy
+, dpkt
+, cffi
+, pandas
+}:
+
+buildPythonApplication rec {
```

Comment:
Applications are not meant to be located within `python-modules`

Please look at awscli as an example  of a python application

Reply:
Uhm Sorry. I forgot to change from 'buildPythonApplication' to 'buildPythonPackage'